### PR TITLE
Update blob_file_size unit MiB to bytes.

### PR DESCRIPTION
**Pull request by WyattJia**
This refers to #612

### CODE REVIEW

**Comment by git-hulk** [comment]
I think this change makes sense since we didn't release them yet, so it won't have the breaking change. cc @ShooterIT

**Comment by tisonkun** [comment]
Is there some background of this change, e.g., a linked issue?

**Comment by git-hulk** [comment]
Thanks for @tisonkun remind, I can add the related issue to explain the background.

File: kvrocks.conf [hunk 1]
```diff
@@ -600,8 +600,8 @@ rocksdb.min_blob_size 4096
 # The size limit for blob files. When writing blob files, a new file is
 # opened once this limit is reached.
 #
-# Default: 256 MiB
-rocksdb.blob_file_size 256
+# Default: 268435456 byte
```
Comment (tisonkun) [inline kvrocks.conf]:
```suggestion
# Default: 268435456 bytes
```

File: src/config.cc [hunk 2]
```diff
@@ -449,26 +449,17 @@ void Config::initFieldCallback() {
       }},
       {"rocksdb.max_bytes_for_level_base", [this](Server* srv, const std::string &k, const std::string& v)->Status {
         if (!srv) return Status::OK();
-        if (!RocksDB.level_compaction_dynamic_level_bytes) {
-          return Status(Status::NotOK, errNotSetLevelCompactionDynamicLevelBytes);
-        }
```
Comment (tisonkun) [inline src/config.cc]: Why do you make these changes?

**Review by tisonkun — COMMENTED** [review]
Generally looks good. Comments inline.

**Comment by ShooterIT** [comment]
I wait this PR long time😂, @git-hulk yes, i agree, we already have thought `blob_file_size` uses byte unit but config file describe wrong unit, we should change it into byte unit.

File: src/config.cc (hunk 2 above)
Comment (WyattJia) [reply src/config.cc]: My bad, this change should be in #610 pull request, I'll clean this change.

**Review by tisonkun — DISMISSED** [review]

**Review by ShooterIT — CHANGES_REQUESTED** [review]
@WyattJia please check all places where we use `blob_file_size`, and make sure not to multiple `MiB`
In `SetBlobDB`, still `RocksDB.blob_file_size * MiB`
```
void Storage::SetBlobDB(rocksdb::ColumnFamilyOptions *cf_options) {
 ...
  cf_options->blob_file_size = config_->RocksDB.blob_file_size * MiB;
 ...
}

**Comment by WyattJia** [comment]
> @WyattJia please check all places where we use `blob_file_size`, and make sure not to multiple `MiB` In `SetBlobDB`, still `RocksDB.blob_file_size * MiB`
> 
> ```
> void Storage::SetBlobDB(rocksdb::ColumnFamilyOptions *cf_options) {
>  ...
>   cf_options->blob_file_size = config_->RocksDB.blob_file_size * MiB;
>  ...
> }
> ```

Thanks for reminding, I've checked unit `about blob_file_size` , @tisonkun @ShooterIT please check and review my changes, extremely grateful!

**Review by tisonkun — APPROVED** [review]

**Review by tisonkun — CHANGES_REQUESTED** [review]
Perhaps `tests/config_test.cc` L74 should be updated from `256` to `268435456` also?

**Comment by WyattJia** [comment]
> Perhaps `tests/config_test.cc` L74 should be updated from `256` to `268435456` also?

Yep, I updated.

**Review by tisonkun — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Review by ShooterIT — APPROVED** [review]

**Comment by ShooterIT** [comment]
Hi @WyattJia as #612 says, we also want to change `max_bytes_for_level_base` into `byte` unit instead of `MiB`, so do you also want to change it?

**Comment by tisonkun** [comment]
@ShooterIT I think it should be covered by #610.

**Comment by ShooterIT** [comment]
oh, yes, thanks @tisonkun

**Comment by tisonkun** [comment]
@WyattJia Thanks for your contribution! Merging...

**Pull request merged**